.github/actions/update-docs/action.yml

```diff
@@ -17,6 +17,10 @@ inputs:
   tag:
     description: "Version tag"
     required: false
+  dev:
+    description: "Should push docs to dev (defaults false)"
+    default: "false"
```

Comment:
I think a name would be more flexible than a boolean, then we can do `dev` and `main` for example